foundation/foundation-mda/src/main/resources/templates/data-access/data.access.pom.xml.vm

```diff
@@ -21,7 +21,7 @@
             <dependency>
                 <groupId>com.boozallen.aissemble</groupId>
                 <artifactId>aissemble-quarkus-bom</artifactId>
-                <version>${project.version}</version>
+                <version>${version.aissemble}</version>
```

Comment:
i: Bug with the data-access pom not having the correct aissemble-quarkus-bom version